lang/gcc7/Portfile

```diff
@@ -233,6 +233,12 @@ if {${subport} eq "libgcc7"} {
         depends_lib-delete   port:zlib
         depends_build-append {*}${depends_lib}
         depends_lib          port:zlib
+    } elseif { ${os.major} < 10 } {
```

Comment:
so we aren’t doing do this skip now.

the decision was that libgcc7 will depend on libgcc8, and then etc to the current libgcc14, just like all the other sustems

Reply:
Yeah, I haven't addressed this yet, as I'm still building gcc14. Will do

Reply:
Done